.env

```
@@ -1 +1,3 @@
 PORT=3000
+YANDEXPRAKTIKUMAPI = "https://ya-praktikum.tech/api/v2/"
```

Довольно тяжело воспринимается такое написание имени переменной. Обычной практикой является использование `_` в качестве разделителя
```suggestion
YANDEX_PRAKTIKUM_API = "https://ya-praktikum.tech/api/v2/"
```